backend/src/status/rooms.ts

```diff
@@ -55,6 +56,28 @@ const getRooms = async () => {
     .innerJoin(buildings, eq(rooms.building, buildings.id));
 };
 
+const addRoom = async (name: string, coordinates: string, building: string): Promise<void> => {
+  const client = await getClient();
```

Comment:
You should not create multiple clients, as this can fill up the client pool as requests come in. You should at least make sure to dispose of it properly after using it.

Reply:
Why not use the global `client` instance instead?